python/setup.py

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2016-2023 Intel Corporation
+# Copyright (c) 2016-2025 Intel Corporation
 #
```

Comment:
Also need to add UXL copyright like in https://github.com/uxlfoundation/oneTBB/commit/cfd170368ec6134696b5794b10de76e90f41abd6#diff-93a059393f5350e47303a2426e2c5390e8c587b8a7533eae2c58cd6b6ed2e9d5R2